src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todos: Todo[],
+  selectTodo: (todo: null) => void,
+  selectedTodoId: number | null,
```

Comment:
don't need render modal if you don’t have selected todo
```suggestion
  selectedTodoId: number,
```